composer.json

```diff
@@ -164,6 +164,7 @@
         "async-aws/core": "<1.5",
         "doctrine/annotations": "<1.13.1",
         "doctrine/dbal": "<2.13.1",
+        "doctrine/event-manager": "<2",
```

Comment:
I wasn't aware that `doctrine/event-manager` is a mandatory dependency. Your adjustments to the `require` section are enough. No need for this conflict rule then.

Reply:
Oh right, updated both composer.json